reporter/diff.js

```diff
@@ -26,7 +26,7 @@
  * Usage: QUnit.diff(expected, actual)
  *
  */
-QUnit.diff = ( function() {
+window.QUnit.diff = ( function() {
```

Comment:
Oh yeah, good catch. This was a remnant from trying to debug a test failure before I realized that the diff tests were running in node.